contributing.md

```diff
@@ -0,0 +1,34 @@
+# Contributing to the handbook
+
```

Comment:
i think it might be worthwhile to use the [OPEN open source introduction](http://openopensource.org/) here:

> Individuals making significant and valuable contributions are given commit-access to the project to contribute as they see fit. This project is more like an open wiki than a standard guarded open source project.

possibly modified like:

> Enspiral contributors making valuable contributions are given permission to contribute as they see fit. This project is more like an open wiki than a guarded project.